AlphaWallet/Share/ShareAction.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ShareAction.swift
+//  AlphaWallet
+//
+//  Created by Vladyslav Shepitko on 11.11.2020.
+//
+
+import Foundation
+
+enum ShareAction: String {
+
+    static let scheme = "awallet"
+
+    case content = "share_content"
```

Comment:
Can you rename most of the references to "share" to "openIn"? Eg. `ShareAction` and "share_content"? A bit more intention revealing.